src/Tribe/Aggregator/Page.php

```diff
@@ -119,6 +119,7 @@ private function __construct() {
 
 		tribe_asset( $plugin, 'tribe-ea-facebook-login', 'aggregator-facebook-login.js', array( 'jquery', 'underscore', 'tribe-dependency' ), 'admin_enqueue_scripts' );
 
+		tribe_notice( 'tribe-aggregator-legacy-import-plugins-active', array( $this, 'notice_legacy_plugins' ), 'type=warning' );
```

Comment:
That's a good question, honestly. The original plan was to avoid permitting the use of Facebook/iCal if the legacy plugins were active. In the interest of simplicity, preventing the notice from being dismissed was a middle ground decision I made. @zbtirrell may have different ideas.